consensus/e2e_tests/state_sync_test.go

```diff
@@ -274,34 +262,33 @@ func TestStateSync_UnsyncedPeerSyncs_Success(t *testing.T) {
 		require.Equal(t, typesCons.NodeId(0), nodeState.LeaderId)
 	}
 
-	unsyncedNode.GetBus().GetConsensusModule().SetAggregatedStateSyncMetadata(uint64(1), testHeight, string(leaderPK.Address()))
+	//unsyncedNode.GetBus().GetConsensusModule().PushStateSyncMetadataResponse(uint64(1), testHeight)
+	metadataReceived := &typesCons.StateSyncMetadataResponse{
+		PeerAddress: "",
```

Comment:
This will make it easier to find if we ever run into an issue.

```suggestion
		PeerAddress: "unused_peer_addr_in_tests",
```

Reply:
Very good point, thanks.